### PR TITLE
#9 Add authorization. Refactor Events

### LINKED ISSUE #19: Authorization
* Simple authorization based on `user.admin?`
* Authorize `[:new, :edit, :create, :update]` controllers actions
* Authorize views by `admin?` helper to show/hide admin's area

### PR DESCRIPTION
**Pull request by sveredyuk**
This PR will close #19

My spec style is little strange, but i found this integration tests very scalable.
I prefer to test units or clickmap only and nothing between.

### CODE REVIEW

**Comment by sveredyuk** [comment]
Review please @kushniryb @illia108 @antonpaisov @sergiy-koshoviy @sergiy-koshoviy @nickKolya @adrenalin23

File: spec/support/features_support.rb [hunk 1]
```diff
@@ -23,3 +23,23 @@ def expect_an_error(hash_pair = {})
   expect(page).to have_css(error_div)
   expect(find("#{error_div} span.error")).to have_content(msg)
 end
+
+# Assume regular user (member) is logged in
+#
+# assume_logged_in
```
Comment (skrypalyk) [inline spec/support/features_support.rb]: do you need this commented code ?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  before_action :authorize_admin!,
+    only:   %i[new edit create update destroy],
+    unless: :devise_controller?
+
+  def authorize_admin!
+    return true if current_user&.admin?
```
Comment (skrypalyk) [inline app/controllers/application_controller.rb]: can we use `admin?` method here ?

File: Gemfile [hunk 3]
```diff
@@ -40,6 +39,7 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'pry-rails'
   gem 'dotenv-rails'
+  gem 'zeus'
```
Comment (skrypalyk) [inline Gemfile]: why do you want to use `zeus` ?

File: spec/support/features_support.rb (hunk 1 above)
Comment (sveredyuk) [inline spec/support/features_support.rb]: this is not commended code, it is comment with example how to use this helper methods

File: Gemfile (hunk 3 above)
Comment (sveredyuk) [inline Gemfile]: `zeus start`
`zeus rspec` - faster spec start...

Comment (skrypalyk) [inline Gemfile]: is it better than `spring` ?

File: spec/features/events/create_spec.rb [hunk 4]
```diff
@@ -0,0 +1,57 @@
+RSpec.describe 'Events features -> Create' do
+  describe 'ACCSESS' do
```
Comment (skrypalyk) [inline spec/features/events/create_spec.rb]: typo in `ACCSESS`

**Pull request merged**